app/src/main/res/layout/activity_main.xml

```diff
@@ -210,6 +210,17 @@
             android:onClick="onPermissionUtilsClicked"
             app:layout_constraintTop_toBottomOf="@id/permissions_utils_text"/>
 
+
+        <Button
+            android:id="@+id/app_diag_activity"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_margin="@dimen/dk_margin_half"
+            android:text="LAUNCH APP DIAGNOSIS"
```

Comment:
@mohamed-garci Text must not be hardcoded, and declared/translated in our dedicated file

Reply:
Done